packages/storybook/src/core/global.scss

```diff
@@ -12,3 +12,16 @@ body {
 .ino-story-composer {
   margin: 16px;
 }
+
+.ino-visually-hidden {
```

Comment:
This is needed for a only visually hidden element (it's still accessible by visually impaired people). We use it for the select menu so that visually impaired people have all the default benefits of the standard select HTML 5 element.

A good description can be found here https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/ it is used in all major frameworks like bootstrap material UI and other libraries.

Reply:
Ahh, nice to know. Thank you!